src/tokens/letter-spacing.yml

```diff
@@ -1,12 +1,12 @@
 #
 # LETTER SPACING TOKENS
-# Use these tokens to adjust letter-spacing.
+# Use these tokens to adjust letter_spacing.
 #
 
 props:
-  letter-spacing-default:
-    value: "-0.15px"
+  letter_spacing_default:
+    value: "_0.15px"
```

Comment:
This looks like an accidental replacement.  this should still be `-0.15px`

Reply:
Good catch! All of the tokens needed to be standardized with underscores in order to work in the HTML, and that line was a fluke. Pushed a fix.